default/web_tt2/help_admin-createlist.tt2

```diff
@@ -1,62 +1,117 @@
+<!-- help_admin-createlist.tt2 -->
+<h2><a name="create_list"></a>[%|helploc%]Requesting the creation of a mailing list[%END%]</h2>
+<p>
+    [%|helploc%]The <strong>list creation request</strong> can be <strong>subject to conditions</strong>. Even though you meet those conditions, the <strong>list creation</strong> will nevertheless be <strong>subject to approval by the listmasters</strong>.[%END%]
+</p>
+<p>
+    [%|helploc%]To request the creation of a mailing list, do as follows:[%END%]
+</p>
+<ol>
+    <li>
+        [%|helploc%]Go to the <strong><a href="../home">list environment homepage</a></strong> and <a href="user-authweb.html"><strong>log on</strong></a>.[%END%]
+    </li>
+    <li>
+        [%|helploc%]In the top menu, <strong>click on the 'Create list' link</strong>.[%END%]
+        <br/>
+        <span class="retraitita">
+            [%|helploc%]If this link does not display, it means that you do not have the privileges required to create a list.[%END%]
+        </span>
+    </li>
+    <li>
+        [%|helploc('',domain)%]Give your list a <strong>name</strong> (only enter the name without the '@' and the domain name; example: <em class="example">languages_spanish</em> and not <em class="example">languages_spanish@%2</em>).[%END%]
+        <br />
+        <span class="retraitita">
+            [%|helploc%]Do not use any spaces, accents or specials characters in list names: those characters might cause problems.[%END%]
+        </span>
+        <span class="retraitita">
+            [%|helploc('',domain)%]Choose an explicit yet short name: think of the subscribers who will have to type this name every time they will send a message to the list! If you manage a set of lists, you can prefix your lists' names with a common prefix; thus they will be sorted together and will be easily recognizable (example: <em class="example">xx-users@%2, xx-hotline@%2</em>, etc.).[%END%]
+        </span>
+    </li>
+    <li>
+        [%|helploc%]Choose a <strong>list type</strong> among the predefined types (the predefined types are only examples of typical configurations that can be changed by the list owners after creation; it is even possible to configure the list beyond the options offered in the list administration module, by asking the listmasters).[%END%]
+    </li>
+    <li>
+        [%|helploc%]Enter a <strong>subject</strong> for your list. This subject will display as a header for all the list pages, and will also be visible on list index pages (list of lists, list of your subscriptions, etc.) and in the browser title bar.[%END%]
+        <br />
+        [%|helploc%]Choose a <strong>topic</strong> in the 'Topics' drop-menu.[%END%]
+        <br />
+        <p class="retraitita">
+            [%|helploc%]If no topic suits your needs, you can request the creation of a new topic by asking the listmasters.[%END%]
+        </p>
+    </li>
```

Comment:
I think this should be:
``` code
    <li>
        [%|helploc%]Enter a <strong>subject</strong> for your list. This subject will display as a header for all the list pages, and will also be visible on list index pages (list of lists, list of your subscriptions, etc.) and in the browser title bar.[%END%]
    </li>
    <li>
        [%|helploc%]Choose a <strong>topic</strong> in the 'Topics' drop-menu.[%END%]
        <p class="retraitita">
            [%|helploc%]If no topic suits your needs, you can request the creation of a new topic by asking the listmasters.[%END%]
        </p>
    </li>
```

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de

Reply:
Confirmed.